dart/lib/src/protocol/sentry_user.dart

```diff
@@ -64,33 +74,65 @@ class SentryUser {
   ///
   /// These keys are stored as extra information but not specifically processed
   /// by Sentry.
+  final Map<String, dynamic>? data;
+
+  @Deprecated('Will be removed in v7. Use [data] instead')
   final Map<String, dynamic>? extras;
 
+  /// Approximate geographical location of the end user or device.
+  ///
+  /// The geolocation is automatically inferred by Sentry.io if the [ipAddress] is set.
+  /// Sentry however doesn't collect the [ipAddress] automatically because it is PII.
+  final SentryGeo? geo;
```

Comment:
```suggestion
  /// The geolocation is automatically inferred by Sentry.io if the [ipAddress] is set.
  /// Sentry however doesn't collect the [ipAddress] automatically because it is PII.
  /// The geo location will currently not be synced to the native layer, if available.
  // See https://github.com/getsentry/sentry-dart/issues/1065
  final SentryGeo? geo;
```